docs/infrastructure/azure/core_infrastructure.md

```diff
@@ -0,0 +1,139 @@
+---
+id: core_infrastructure
+title: Azure Core Infrastructure
+sidebar_label: Core Infrastructure
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The core infrastructure is the foundation for all other Stacks Workloads. As, in most cases, this will be the first part of Stacks that you deploy we will also cover bootstrapping your Azure tenant.
+
+## Resources Provisioned
+
+### Diagram
+
+<img alt="Azure Core Infrastructure" src={useBaseUrl('img/azure_core_infrastructure.png')} />
+
+### Resource List
+
+| Resource                  | Description                                                          |
+| ------------------------- | -------------------------------------------------------------------- |
+| Resource Group            | Used to logically group infrastructure                               |
+| Virtual Network           | Fundamental building block for the private network                   |
+| Subnet - AGW              | Dedicated subnet required for Application Gateway                    |
+| Application Gateway       | Web traffic load balancer                                            |
+| Public IP                 | IP address associated with the Application Gateway                   |
+| Subnet - k8s              | Subnet used by the AKS cluster                                       |
+| AKS                       | Azure Kubernetes Service                                             |
+| Resource Group: Node Pool | AKS created resource group for nodes, load balancers, etc.           |
+| Internal Load Balancer    | Load balancer used with NGINX ingress                                |
+| VM Scale Set: Nodes       | Virtual Machine scaling for AKS                                      |
+| NGINX Ingress             | Nginx ingress Kubernetes namespace, deployment and service           |
+| Internal DNS Zone         | Custom DNS domain                                                    |
+| Public DNS Zone           | Hosted service for DNS domain                                        |
+| Application Insights      | Application performance management, monitoring and analytics service |
+| Log Analytics Workspace   | Log analytics with container insights                                |
+| Managed Identity          | Managed Identity with aadpodidentity binding                         |
```

Comment:
Just a note here, the pod identity used in Stacks is deprecated, we need to replace it with their newer preview instead.

Reply:
Nice one! I'll get a ticket in the backlog for that so it's on everyone's radar